scripts/circle-ci/publish-to-antithesis.sh

```diff
@@ -20,11 +30,16 @@ docker pull palantirtechnologies/cassandra:2.2.18-1.112.0-rc5
 docker tag palantirtechnologies/cassandra:2.2.18-1.112.0-rc5 ${ANTITHESIS_REPO_URL}/cassandra:2.2.18-1.112.0-rc5
 docker push ${ANTITHESIS_REPO_URL}/cassandra:2.2.18-1.112.0-rc5
 
-docker tag palantirtechnologies/timelock-server-distribution:${VERSION} ${ANTITHESIS_REPO_URL}/timelock-server-distribution:${VERSION}
-docker push ${ANTITHESIS_REPO_URL}/timelock-server-distribution:${VERSION}
+docker tag palantirtechnologies/timelock-server-distribution:${VERSION} ${ANTITHESIS_REPO_URL}/timelock-server-distribution:${1}
+docker push ${ANTITHESIS_REPO_URL}/timelock-server-distribution:${1}
+
+docker tag palantirtechnologies/atlasdb-workload-server-distribution:${VERSION} ${ANTITHESIS_REPO_URL}/atlasdb-workload-server-distribution:${1}
+docker push ${ANTITHESIS_REPO_URL}/atlasdb-workload-server-distribution:${1}
 
-docker tag palantirtechnologies/atlasdb-workload-server-distribution:${VERSION} ${ANTITHESIS_REPO_URL}/atlasdb-workload-server-distribution:${VERSION}
-docker push ${ANTITHESIS_REPO_URL}/atlasdb-workload-server-distribution:${VERSION}
+docker tag palantirtechnologies/atlasdb-workload-server-antithesis:${VERSION} ${ANTITHESIS_REPO_URL}/atlasdb-workload-server-antithesis:${1}
+docker push ${ANTITHESIS_REPO_URL}/atlasdb-workload-server-antithesis:${1}
```

Comment:
question: Is there a difference between `$1` and `${1}`? Haven't seen this notation before, though pretty sure it works too

Reply:
my bash knowledge is fairly poor tbh, but in this context I believe they are interchangeable? I think `${1}` is typically used for templating in strings, as `$1` would just render as one dollar. Not sure why I went with this way first time around!